@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1355,7 +1355,7 @@ export class TextElementsRenderer {
             placementStats.log();
         }
 
-        if (!this.m_options.disableFading && renderParams.fadeAnimationRunning) {
+        if (renderParams.fadeAnimationRunning) {
```

Comment:
This was changed because with the current implementation of fading in `RenderState` it still takes 1 frame for the fading animation to complete when fading is disabled(`updateFading` needs to be called once). This caused some of the labels to not be visible with `MapView.disableFading=true`.